Lib/ctypes/__init__.py

```diff
@@ -296,7 +299,7 @@ def create_unicode_buffer(init, size=None):
         return buf
     elif isinstance(init, int):
         _sys.audit("ctypes.create_unicode_buffer", None, init)
-        buftype = c_wchar * init
+        buftype = c_wchar.__mul__(init)
```

Comment:
Is this also local patch? Then,

```suggestion
        # XXX: RustPython
        # buftype = c_wchar * init
        buftype = c_wchar.__mul__(init)
```